components/autocomplete/index.js

```diff
@@ -176,40 +351,76 @@ class Autocomplete extends Component {
 		// Any handled keycode should prevent original behavior. This relies on
 		// the early return in the default case.
 		event.preventDefault();
-		event.stopImmediatePropagation();
+		event.stopPropagation();
+	}
+
+	toggleKeyEvents( isListening ) {
+		// This exists because we must capture ENTER key presses before Editable.
```

Comment:
If enter works correctly in master with the block autocomplete, why do we suddenly need this now?

Reply:
The enter key handler in master relied on overriding the keyhandler of the Editable. I'm not certain why it was able to get the event before TinyMCE when a onKeyDownCapture handler on the surrounding component was not - I guess the react simulated events don't mix with native events perfectly. When I was initially working on nesting these autocomplete handlers that approach was unworkable though now it is only one component I suppose we could return to it. That said I think that overriding the event handler on a child block is very weird and I much prefer this approach.